src/lib/containers/synapse_table_functions/GetColumnIndiciesWithType.ts

```diff
@@ -0,0 +1,16 @@
+import { EntityColumnType, QueryResultBundle } from '../../utils/synapseTypes/'
+
+export const getColumnIndiciesWithType = (
+  data: QueryResultBundle | undefined,
+  ...columnTypes: EntityColumnType[]
+) => {
+  const columnsOfTypeEntity: number[] = []
+  data &&
```

Comment:
now that we have typescript conditional properties we can clean this up a bit :sunglasses:
```
data?.selectColumns?.forEach((el, index) => {
      if (columnTypes.includes(el.columnType)) {
        columnsOfTypeEntity.push(index)
      }
    })
```

Reply:
Wow this is literally super nice catch!